generator/templates/Default/src/api/implementation/app/index.js

```diff
@@ -32,13 +31,12 @@ const instance = axios.create(config);
 instance.interceptors.request.use(onRequestFulFilled, onRequestRejected);
 instance.interceptors.response.use(onResponseFulFilled, onResponseRejected);
 
-const get = instance.get;
-const post = instance.post;
+const { get } = instance;
+const { post } = instance;
 const destroy = instance.delete;
-const put = instance.put;
+const { put } = instance;
```

Comment:
Kan je niet gewoon het volgende doen:

```
const { get, post, destroy, put } = instance;
```

Reply:
> Kan je niet gewoon het volgende doen:
> 
> ```
> const { get, post, destroy, put } = instance;
> ```

Ja en nee. Als ik Ricardo goeddd begrijp is destroy geen key van de instance. 

`const { get, post, delete: destroy, put } = instance` wordt het dan eventueel.